api/buildcraft/api/transport/pipe/PipeApi.java

```diff
@@ -135,7 +134,7 @@ public PowerTransferInfo(long transferPerTick, long lossPerTick, long resistance
     @CapabilityInject(IInjectable.class)
     private static Capability<IInjectable> capInjectable;
 
-    static {
+    public static void initCapabilites() {
```

Comment:
final was removed because this wasn't a static block anymore, i would need to take another look if it is still needed, early in the port there where some issues with the mod lifecycle that ment these where not getting initialized in time

Reply:
Isn't there an annotation for that?

Reply:
there is (see above) but for some reason BC is doing a safety check that it actually registered (though i don't remember registering ever being able to fail, need to look into that)